ardupilot/equipment/power/20009.BatteryPeriodic.uavcan

```diff
@@ -0,0 +1,29 @@
+#
+# Battery data to be sent statically upon request or periodically at a low rate
+#
+
+uint8[<=50] name                       # Formatted as manufacturer_product, 0 terminated
+uint8[<=16] serial_number              # Serial number in ASCII characters, 0 terminated
```

Comment:
Just reviewing a new dronecan battery coming out and they use a pack code that is 16 chars for the serial number. Maybe we should make it slightly bigger? Also if we don't specify that when full it is not zero terminated  then we need at minimum 17 chars

```suggestion
uint8[<=24] serial_number              # Serial number in ASCII characters, 0 terminated
```

Reply:
Historically it was 16 chars because the TI chips have a register for it that is 16 chars. IMO if we don't have an example of something bigger than 16 we should leave it.

Reply:
I mostly agree with you I just hate being foolish with bits or bytes on a message that should be sent out once....  Or every few minutes...

Reply:
Oh yeah this is the periodic message, may as well make it 32?

Reply:
Sure make it 32. that was my first thought anyways :)